EssentialEstablishmentGenerator/GeneralStore/JS/GeneralStoreRenders.js

```diff
@@ -1,101 +1,83 @@
-setup.GeneralStoreRenders = function (GeneralStore) {
-  var warmthRoll = random(1, 100)
-
-  if (GeneralStore.roll.size > 80) {
-    GeneralStore.size = 'huge'
-    warmthRoll -= 20
-  } else if (GeneralStore.roll.size > 70) {
-    GeneralStore.size = 'quite large'
-    warmthRoll -= 15
-  } else if (GeneralStore.roll.size > 60) {
-    GeneralStore.size = 'large'
-    warmthRoll -= 10
-  } else if (GeneralStore.roll.size > 50) {
-    GeneralStore.size = 'spacious'
-    warmthRoll -= 5
-  } else if (GeneralStore.roll.size > 40) {
-    GeneralStore.size = 'medium'
-  } else if (GeneralStore.roll.size > 30) {
-    GeneralStore.size = 'slightly cramped'
-    warmthRoll += 15
-  } else if (GeneralStore.roll.size > 20) {
-    GeneralStore.size = 'small'
-    warmthRoll += 15
-  } else if (GeneralStore.roll.size <= 20) {
-    GeneralStore.size = 'tiny'
-    warmthRoll += 30
+setup.GeneralStoreRenders = function(GeneralStore) {
+  let warmthRoll = random(1, 100);
+  const size = GeneralStore.size;
+  if (size > 80) {
+    GeneralStore.size = "huge";
+    warmthRoll -= 20;
+  } else if (size > 70) {
+    GeneralStore.size = "quite large";
+    warmthRoll -= 15;
+  } else if (size > 60) {
+    GeneralStore.size = "large";
+    warmthRoll -= 10;
+  } else if (size > 50) {
+    GeneralStore.size = "spacious";
+    warmthRoll -= 5;
+  } else if (size > 40) {
+    GeneralStore.size = "medium";
+  } else if (size > 30) {
+    GeneralStore.size = "slightly cramped";
+    warmthRoll += 15;
+  } else if (size > 20) {
+    GeneralStore.size = "small";
+    warmthRoll += 15;
+  } else if (size <= 20) {
+    GeneralStore.size = "tiny";
+    warmthRoll += 30;
   }
 
-  if (warmthRoll > 80) {
-    GeneralStore.warmth = 'swelteringly hot'
-  } else if (warmthRoll > 70) {
-    GeneralStore.warmth = 'extremely warm'
-  } else if (warmthRoll > 60) {
-    GeneralStore.warmth = 'uncomfortably warm'
-  } else if (warmthRoll > 50) {
-    GeneralStore.warmth = 'nice and toasty'
-  } else if (warmthRoll > 40) {
-    GeneralStore.warmth = 'quite warm'
-  } else if (warmthRoll > 30) {
-    GeneralStore.warmth = 'warm'
-  } else if (warmthRoll > 20) {
-    GeneralStore.warmth = 'mild'
-  } else if (warmthRoll <= 20) {
-    GeneralStore.warmth = 'cold'
-  }
+  const attributes = {
```

Comment:
Bug fix: need to add `GeneralStore.roll.warmth = warmthRoll;` right before the attributes assignment